snippets/findify-product-card.liquid

```diff
@@ -1,37 +1,37 @@
+    class="findify-product-card {{ additional_class }}"
     data-product-id={{ card_product.id }}
     data-variant-id={{ card_product.id }}
     data-product-url={{ card_product.url }}
-    data-findify-product-card
-    class="card-wrapper product-card-wrapper underline-links-hover"
+    findify-widget-type={{ widget_type }}
+    href={{ card_product.url }}
+    
   >
-    <div class="image-wrapper" style="position: relative; width: 100%">
-      <img
-        style="position: relative; width: 100%; height: auto"
+    <a href={{ card_product.url }} >
+      <img  
+        src="{{ card_product.featured_media | image_url: width: 533 }}" 
         srcset="
           {%- if card_product.featured_media.width >= 165 -%}{{ card_product.featured_media | image_url: width: 165 }} 165w,{%- endif -%}
           {%- if card_product.featured_media.width >= 360 -%}{{ card_product.featured_media | image_url: width: 360 }} 360w,{%- endif -%}
           {%- if card_product.featured_media.width >= 533 -%}{{ card_product.featured_media | image_url: width: 533 }} 533w,{%- endif -%}
           {%- if card_product.featured_media.width >= 720 -%}{{ card_product.featured_media | image_url: width: 720 }} 720w,{%- endif -%}
           {%- if card_product.featured_media.width >= 940 -%}{{ card_product.featured_media | image_url: width: 940 }} 940w,{%- endif -%}
           {%- if card_product.featured_media.width >= 1066 -%}{{ card_product.featured_media | image_url: width: 1066 }} 1066w,{%- endif -%}
-          {{ card_product.featured_media | image_url }} {{ card_product.featured_media.width }}w
-        "
-        src="{{ card_product.featured_media | image_url: width: 533 }}"
-        sizes="(min-width: {{ settings.page_width }}px) {{ settings.page_width | minus: 130 | divided_by: 4 }}px, (min-width: 990px) calc((100vw - 130px) / 4), (min-width: 750px) calc((100vw - 120px) / 3), calc((100vw - 35px) / 2)"
-        alt="{{ card_product.featured_media.alt | escape }}"
-        class="motion-reduce"
-        {% unless lazy_load == false %}
-          loading="lazy"
-        {% endunless %}
+          {{ card_product.featured_media | image_url }} {{ card_product.featured_media.width }}
+          "
+        alt="{{ card_product.featured_media.alt | escape }}" 
+        class="adapt" 
         width="{{ card_product.featured_media.width }}"
-        height="{{ card_product.featured_media.height }}"
+        height="{{ card_product.featured_media.height }}" 
+        style="position: relative; width: 100%; height: auto"
+        loading="lazy"
       />
-    </div>
-    <div class="content-wrapper">
-      <div class='findify-product-title'>{{ card_product.title }}</div>
-      {% render 'findify-product-price', product: card_product %}
-    </div>
- </div>
-{%- endif -%}
+      <div class="content-wrapper">
+        <div class='findify-product-title findify-cut-text'>{{ card_product.title }}</div>
+        {% render 'findify-product-price', product: card_product %}
```

Comment:
Ah, I realised you didn't change the price.liquid

Could you switch the prices? As commented within the ticket?